docs/conf.py

```diff
@@ -78,14 +78,14 @@
 # |version| and |release|, also used in various other places throughout the
 # built documents.
 
-__import__(setup_cfg['package_name'])
-package = sys.modules[setup_cfg['package_name']]
-
-# The short X.Y version.
-version = package.__version__.split('-', 1)[0]
-# The full version, including alpha/beta/rc tags.
-release = package.__version__
+from pkg_resources import get_distribution, DistributionNotFound
+try:
+    __version__ = get_distribution(__name__).version
```

Comment:
This isn't going to work, you should use something like [this](https://github.com/sunpy/sunpy/blob/master/docs/conf.py#L99) as suggested in the [setuptools_scm](https://github.com/pypa/setuptools_scm#usage-from-sphinx)